src/main/java/io/github/jhipster/online/web/rest/GitResource.java

```diff
@@ -161,30 +161,30 @@ public RedirectView callback(@PathVariable String gitProvider, String code) {
 
     public static class GitAccessTokenRequest {
 
-        private String client_id;
+        private String clientId;
 
-        private String client_secret;
+        private String clientSecret;
```

Comment:
Did you test generating an application with these changes?  Some OAuth providers expect the parameters to be `client_id` and `client_secret` instead of `clientId` and `clientSecret` ([Github OAuth Docs](https://docs.github.com/en/free-pro-team@latest/developers/apps/authorizing-oauth-apps#parameters-1))

We may be able to add an annotation such as `@JsonProperty("client_id")`, did not test this change.

Reply:
arf you're right

Reply:
I remove this commit